hubble/executor/hubio.py

```diff
@@ -456,6 +456,11 @@ def _sync_push(
             subject = stream_msg.get('subject')
             payload = stream_msg.get('payload', '')
 
+            # authentication error doesn't follow the streaming format
+            if stream_msg.get('code', 200) >= 400:
+                msg = stream_msg.get('readableMessage')
+                raise Exception(f'{ msg or "Unknown Error"} session_id: {session_id}')
+
```

Comment:
Better check this before resp.iter_lines?

If there's an authentication error the `resp.status_code` will be 401 instead of 200